x-pack/plugins/ml/public/application/data_frame_analytics/common/fields.ts

```diff
@@ -180,24 +180,22 @@ export const getDefaultFieldsFromJobCaps = (
   // default is 'ml'
   const resultsField = jobConfig.dest.results_field;
 
-  const featureImportanceFields = [];
-  const topClassesFields = [];
   const allFields: any = [];
   let type: ES_FIELD_TYPES | undefined;
   let predictedField: string | undefined;
 
   if (isOutlierAnalysis(jobConfig.analysis)) {
-    if (jobConfig.analysis.outlier_detection.compute_feature_influence) {
-      featureImportanceFields.push({
-        id: `${resultsField}.${FEATURE_INFLUENCE}`,
-        name: `${resultsField}.${FEATURE_INFLUENCE}`,
-        type: KBN_FIELD_TYPES.UNKNOWN,
-      });
+    if (!jobConfig.analysis.outlier_detection.compute_feature_influence) {
+      // remove all feature influence fields
+      fields = fields.filter(
+        (field: any) => !field.name.includes(`${resultsField}.${FEATURE_INFLUENCE}`)
```

Comment:
Can we replace the use of `any` in this file?

Reply:
Updated in fa3e81bdcbfd5b0fe270287619220c7907a36a59